pkg/forward/forward_test.go

```diff
@@ -259,7 +319,11 @@ func TestNewInterStepDataForward(t *testing.T) {
 			otKeys2, _ := otStores["to2"].GetAllKeys(ctx)
 			otValue2, _ := otStores["to2"].GetValue(ctx, otKeys2[0])
 			otDecode2, _ := wmb.DecodeToWMB(otValue2)
-			assert.False(t, otDecode2.Idle)
+			if tt.batchSize > 1 {
+				assert.False(t, otDecode2.Idle)
```

Comment:
Do you mind adding a comment on why `batchSize <= 1`  then `otDecode2.Idle` would be true?

Reply:
done

Reply:
done